lib/file_transfer_agent/encrypt_util.js

```diff
@@ -226,7 +227,7 @@ function EncryptUtil(encrypt, filestream, temp) {
     await new Promise((resolve) => {
```

Comment:
It could be like new Promise((resolve, reject) => ...) without require('async')

Reply:
Done